<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BML: Bayesan Mutation Landscape</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
          </property>
          <property name="title">
           <string>Input/Output management </string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Job ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="jobLine">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Data Matrix' Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="matrixPathLine">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="styleSheet">
             <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
            </property>
            <property name="title">
             <string>Performance settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Number of threads to be used</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="numberOfThreadsLine">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="styleSheet">
             <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
            </property>
            <property name="title">
             <string>Matrix preprocessor settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Cutoff</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="CutoffLine">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Disabled</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="autoCutoffCheckBox">
                 <property name="text">
                  <string>Auto compute</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <property name="tristate">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="styleSheet">
           <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
          </property>
          <property name="title">
           <string>Elaboration options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Number of tree reseeds</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="reseedsLine">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Threshold for inferring paths</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="thresholdLine">
               <property name="text">
                <string>0.3</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Number of bootstrap replicates</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="replicatesLine">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Disabled</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBootstrap">
               <property name="text">
                <string>Enable bootstrap</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="styleSheet">
         <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
        </property>
        <property name="title">
         <string>Informations:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPlainTextEdit" name="logText">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QPushButton" name="mafButton">
        <property name="text">
         <string>Run MAF file preprocessor utility</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Execute BML analysis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGuide"/>
    <addaction name="actionChangelog"/>
    <addaction name="actionAbout_BML"/>
    <addaction name="actionBML_publication"/>
    <addaction name="separator"/>
    <addaction name="actioncBioPortal"/>
    <addaction name="actionGraphviz_Online"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_BML">
   <property name="text">
    <string>About BML</string>
   </property>
  </action>
  <action name="actionGuide">
   <property name="text">
    <string>Guide</string>
   </property>
  </action>
  <action name="actionBML_publication">
   <property name="text">
    <string>BML publication</string>
   </property>
  </action>
  <action name="actionChangelog">
   <property name="text">
    <string>Changelog</string>
   </property>
  </action>
  <action name="actionGraphviz_Online">
   <property name="text">
    <string>Graphviz Online</string>
   </property>
  </action>
  <action name="actioncBioPortal">
   <property name="text">
    <string>cBioPortal datasets</string>
   </property>
  </action>
  <action name="actionBML_supplementary_data">
   <property name="text">
    <string>BML supplementary data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbout_BML</sender>
   <signal>triggered()</signal>
   <receiver>MainMenu</receiver>
   <slot>openHelp_About_BML()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBML_publication</sender>
   <signal>triggered()</signal>
   <receiver>MainMenu</receiver>
   <slot>openHelp_BML_publication()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChangelog</sender>
   <signal>triggered()</signal>
   <receiver>MainMenu</receiver>
   <slot>openHelp_Changelog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGraphviz_Online</sender>
   <signal>triggered()</signal>
   <receiver>MainMenu</receiver>
   <slot>openHelp_Graphviz()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGuide</sender>
   <signal>triggered()</signal>
   <receiver>MainMenu</receiver>
   <slot>openHelp_Guide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>pressed()</signal>
   <receiver>MainMenu</receiver>
   <slot>browseMatrixPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1455</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoCutoffCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainMenu</receiver>
   <slot>cutoffAutoSet(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1379</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>1524</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBootstrap</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainMenu</receiver>
   <slot>bootstrapEnabledSet(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1490</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>1521</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>clicked()</signal>
   <receiver>MainMenu</receiver>
   <slot>runBML()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1491</x>
     <y>960</y>
    </hint>
    <hint type="destinationlabel">
     <x>1524</x>
     <y>952</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mafButton</sender>
   <signal>clicked()</signal>
   <receiver>MainMenu</receiver>
   <slot>runMAF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>976</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>989</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioncBioPortal</sender>
   <signal>triggered()</signal>
   <receiver>MainMenu</receiver>
   <slot>openHelp_cBioPortal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openHelp_About_BML()</slot>
  <slot>openHelp_Changelog()</slot>
  <slot>openHelp_BML_publication()</slot>
  <slot>openHelp_Graphviz()</slot>
  <slot>openHelp_Guide()</slot>
  <slot>openHelp_cBioPortal()</slot>
  <slot>browseMatrixPressed()</slot>
  <slot>cutoffAutoSet(bool)</slot>
  <slot>bootstrapEnabledSet(bool)</slot>
  <slot>runBML()</slot>
  <slot>runMAF()</slot>
  <slot>openHelp_sup_data()</slot>
 </slots>
</ui>
